finetune/base_models/bert/featurizer.py

```diff
@@ -119,3 +121,24 @@ def bert_featurizer(
             output_state = {k: tf.stop_gradient(v) for k, v in output_state.items()}
 
         return output_state
+
+def bert_featurizer_with_pos(
+    X,
+    context,
+    encoder,
+    config,
+    train=False,
+    reuse=None,
+    **kwargs
+):
+    featurizer_state = bert_featurizer(X, encoder, config, train=train, reuse=reuse, **kwargs)
+    with tf.variable_scope("model/featurizer"):
+        embed_position(context, featurizer_state, config, train)
+        add_context_embed(featurizer_state)
+        hidden = featurizer_state['sequence_features']
+        w0, w = smooth_pos_attn(hidden, config, featurizer_state['lengths'])
+        text_embed = hidden[:, :, :config.n_embed]
+        seq_feats = tf.matmul(w, text_embed)
+        featurizer_state['sequence_features'] = seq_feats
```

Comment:
This doesn't seem to give us correct classification features, can we either explicitly remove them or set them correctly

Reply:
yeah whoops I meant to add a warning but forgot